src/Common/src/System/Security/Cryptography/Asn1/AsnXml.targets

```diff
@@ -0,0 +1,43 @@
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <AsnXml />
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="$(MSBuildThisFileDirectory)asn.xsd">
+      <Link>Common\System\Security\Cryptography\Asn1\asn.xsd</Link>
+    </None>
+  </ItemGroup>
+
+  <!-- MSBuild for .NET Core doesn't support XslTransform at this time -->
+  <Target Name="CompileAsn" BeforeTargets="CoreCompile"
+    Condition="'$(RunningOnCore)' != 'true'"
+    Inputs="@(AsnXml);$(MSBuildThisFileDirectory)asn.xslt"
+    Outputs="%(Identity).cs">
+
+    <MakeDir Directories="$(IntermediateOutputPath)asnxml" />
+
+    <XslTransformation
+      XslInputPath="$(MSBuildThisFileDirectory)asn.xslt"
+      XmlInputPaths="@(AsnXml)"
+      OutputPaths="@(AsnXml -> '$(IntermediateOutputPath)asnxml\%(filename).cs')" />
+
+    <Exec Condition="'$(RunningOnUnix)' != 'true'"
+      IgnoreExitCode="true"
+      StandardOutputImportance="Low"
+      Command="$(SystemRoot)\System32\fc.exe /a @(AsnXml -> '$(IntermediateOutputPath)asnxml\%(filename).cs') @(AsnXml -> '%(Identity).cs')">
+      <Output TaskParameter="ExitCode" ItemName="_AsnXmlDiffCode" />
+    </Exec>
+
+    <!-- TODO: Call diff on Unix -->
+    <ItemGroup Condition="'$(RunningOnUnix)' == 'true'">
+      <_AsnXmlDiffCode Include="1" />
+    </ItemGroup>
```

Comment:
I don't get this. Is it intentional that the comparison is not performed on .NET Core?